.eslintignore

```diff
@@ -5,3 +5,6 @@ coverage
 tests
 src/worker.ts
 src/web_worker_kernel.ts
+
+# TODO: remove
```

Comment:
Ah I missed this one.

Happy to track it in a separate issue to not block this PR if that's fine.